client/src/legacy/HtmlEditorField.js

```diff
@@ -83,6 +86,7 @@ ss.editorWrappers.tinyMCE = (function() {
     getConfig: function() {
       var selector = "#" + editorID,
         config = jQuery(selector).data('config'),
+        height = config.row_height ? config.row_height : 'undefined',
```

Comment:
```suggestion
        height = config.row_height ? config.row_height : undefined,
```
The `undefined` type must not have quotes, or else it'll be `typeof string`

Reply:
Updated